test/basic.test.js

```diff
@@ -344,3 +344,26 @@ test('render a single quote as JSON', (t) => {
   t.equal(output, JSON.stringify(toStringify))
   t.ok(validate(JSON.parse(output)), 'valid schema')
 })
+
+test('should not break when the value of field is null', (t) => {
+  t.plan(2)
+
+  const schema = {
+    title: 'an object in a string',
+    type: 'object',
+    properties: {
+      updatedAt: {
+        type: ['string'],
+        format: 'date-time'
+      }
+    }
+  }
+
+  const toStringify = { updatedAt: null }
+  const validate = validator(schema)
+  const stringify = build(schema)
+  const output = stringify(toStringify)
+
+  t.equal(output, JSON.stringify(toStringify))
```

Comment:
```suggestion
  t.equal(output, '{"updatedAt":""}')
```

When there is a type `string` not nullable, we must return an empty string.

Reply:
@Eomm Are you sure that's the right behavior?
expected: string, given: null, output: "" or null ?
expected: date, given: null, output: <blind date> or null ? :)
expected: bool, given: null, output: false or null

Reply:
null is a type in the json schema, so it will never be returned if you don't want it.

date, is a format, so the right behaviour is to return the default string when it is null: an empty string

Reply:
yep for a schema like this, we can return an empty string

```javascript
updatedAt: {
  type: 'string',
  format: 'date-time'
}
```

but for 

```javascript
updatedAt: {
  type: ['string'],
  format: 'date-time'
}
```

it will return `null` ref https://github.com/fastify/fast-json-stringify/blob/1241c5d2091436200210df9404f64a82a3af1a8e/index.js#L1113-L1114

Reply:
Yes, this was wrong for sure!